src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -296,4 +296,14 @@ public void installPlugin(File localFile) {
         }
         return shortNamesVersion;
     }
+
+    /**
+     * Enable oder disable the specified plugin.
+     * @param spec plugin id (e.g. "ldap")
+     * @param state enable plugin if true, disable plugin if false
+     */
+    public void enablePlugin(PluginSpec spec, boolean state) {
```

Comment:
It is part of the PluginManager.

So it is simpler to use the parameter `String pluginName`?